plan.go

```diff
@@ -2837,17 +2839,20 @@ func (r *emptyFieldsPlan) do(ctx *execCtx, f func(id interface{}, data []interfa
 
 type wrapFilterPlan struct {
 	*filterDefaultPlan
+	once  sync.Once
```

Comment:
> One question though, I dont see the ned to guard the map wih sync.RWMutex from your explanation it doesnt seem like there will be any concurrent use of the same execCtx instance.

Yep, I believe execCtx is not used concurrently. However, It's always a good idea to run the tests with -race after changes like this one.